ts/session/messages/outgoing/content/data/group/ClosedGroupLeaveMessage.ts

```diff
@@ -0,0 +1,30 @@
+import { SignalService } from '../../../../../../protobuf';
+import {
+  ClosedGroupMessage,
+  ClosedGroupMessageParams,
+} from './ClosedGroupMessage';
+
+export class ClosedGroupLeaveMessage extends ClosedGroupMessage {
+  constructor(params: ClosedGroupMessageParams) {
+    super({
+      timestamp: params.timestamp,
+      identifier: params.identifier,
+      groupId: params.groupId,
+    });
+  }
+
+  public ttl(): number {
+    return this.getDefaultTTL();
+  }
+
+  protected groupContextType(): SignalService.GroupContext.Type {
+    return SignalService.GroupContext.Type.QUIT;
+  }
+
+  protected dataProto(): SignalService.DataMessage {
+    const messageProto = new SignalService.DataMessage();
```

Comment:
`ClosedGroupMessage` already implements this

Reply:
copy paste mistake